libraries/iio/iio.h

```diff
@@ -0,0 +1,98 @@
+/***************************************************************************//**
```

Comment:
do we want to have duplicate files?
https://github.com/analogdevicesinc/no-OS/blob/master/iio/iio.h